airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -51,6 +54,14 @@ public class PostgresQueryUtils {
             txid_snapshot_xmin(txid_current_snapshot()) AS xmin_raw_value;
       """;
 
+  public static final String CTID_FULL_VACUUM_IN_PROGRESS =
+      """
+      SELECT phase FROM pg_stat_progress_cluster WHERE relid=to_regclass('%s')::oid    
```

Comment:
Quick question : This seems different from the query in the tech spec. What is the difference between the two queries?

Reply:
This filters only for rows related to this stream. I'll add also `AND command = 'VACUUM FULL'`